Tasks/CopyFilesOverSSHV0/copyfilesoverssh.ts

```diff
@@ -1,8 +1,7 @@
 import * as os from 'os';
 import * as path from 'path';
-
 import * as tl from 'vsts-task-lib/task';
-import { SshHelper } from 'ssh-common/ssh-common';
+import { SshHelper } from './ssh';
```

Comment:
While I don't think this will have any functional change, I think we should still bump the version for the task.